gulpfile.js

```diff
@@ -9,7 +9,7 @@ const replace = require('gulp-replace');
 const rename = require('gulp-rename');
 const gulpif = require('gulp-if');
 const sass = require('gulp-sass')(require('sass'));
-const gulpStylelint = require('gulp-stylelint');
```

Comment:
`gulp-stylelint` may be abandoned, as it doesn't support the newest major release of Stylelint released early in the year, and the maintainer hasn't been active in related issues (https://github.com/olegskl/gulp-stylelint/issues/132).

In any case, the cumulative dependency shifting here removed _a lot_ from the dependency tree (net -1859 lines in `package-lock.json`), which I'm pretty happy about.